### DISCUSSION TITLE
迁移 wiki 内容至仓库本身的提议

### DISCUSSION THREAD

**Discussion by KimYangOfCat** [FEDarling/fe-translate-weekly#167] [项目讨论]
之前关于项目的信息以及合作方式的简单说明均写在了仓库自带的 wiki 中，经过我个人的思索，我认为目前仓库自带的 wiki 功能有如下两点问题：
1. 对图片支持不好，需要图床
2. wiki 本身也是一个git仓库，但是一个独立的仓库而与主仓库隔绝，导致很多内容信息与主仓库的内容不同步
这两点问题，促使我希望能将wiki内容合并进入主仓库，能直接存储静态图片，不需要图床，合并为一个仓库也能减少维护时的精力分散，提升相关信息的准确度。

目前我已新建了 wiki 文件夹，并在其中简要说明了接下来需要写的一些wiki 文档，请参考[wiki](https://github.com/FEDarling/weekly-tracker/tree/main/weeklys/wiki)
不知道大家对这个提议有没有什么想法？ @daodaolee @LilyChenlin

**Comment by daodaolee** [comment]
1. wiki的话，可以合并，这个没什么问题
2. 图片等资源的话，我还是建议图床，因为一旦时间长了，本地的图片多了的话，打包的时长会成倍增加，github本身是支持图床功能的，gitee也可以，而且有工具也很方便

**Reply by KimYangOfCat** [reply]
wiki 文档里的图片数量不多，不会频繁变更，所以我建议直接放在本地，至于打包时长，目前打包工作由action自动进行，时长问题似乎不必考虑？

使用图床当然也是可以哒，只是使用公有图床会有图片资源丢失的风险，使用私有图床会有一定经济成本，且不利于项目成员共同使用。

使用github图床和直接存放在本地，其实国内访问情况其实差不多，只是方便分发markdown文件，但是本次只讨论wiki，wiki 文档不需要分发行为，所以关于 wiki 的图片资源，我认为直接存放在仓库中更省事儿，不必专门新建一个仓库用于存储图片

至于译文中的图片资源，暂时就保持原链吧，特殊情况考虑仓库存储。

**Reply by daodaolee** [reply]
如果纯wiki的话，那倒没啥，完全可以放本地！😄
